docs/CONTRIBUTING.md

```diff
@@ -1,90 +1,132 @@
-# Contribution Guidelines
+# Gitbase Playground Contribution Guidelines
 
 As all source{d} projects, this project follows the
 [source{d} Contributing Guidelines](https://github.com/src-d/guide/blob/master/engineering/documents/CONTRIBUTING.md).
 
+In addition to those guidelines, this document describes how to build and run the application for development purposes.
 
-## Additional Contribution Guidelines
+# Requirements
 
-In addition to the [source{d} Contributing Guidelines](https://github.com/src-d/guide/blob/master/engineering/documents/CONTRIBUTING.md),
-this project follows the guidelines described below.
+Refer to the main [README](../README.md) for the common requirements and configuration environment variables. As with a normal deployment, if you don't use Docker Compose you will need to configure a [gitbase](https://docs.sourced.tech/gitbase) server, and a [bblfsh](https://doc.bblf.sh/) server.
```

Comment:
`configure server manually, as in <link to Docker dependency section>`

Reply:
imo it's up to user use docker for deps or not

Reply:
I put a new subsection here in 1827f43.